test/functional/update/update-no-minversion.bats

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env bats
+
+load "../testlib"
+
+test_setup() {
+
+	create_test_environment "$TEST_NAME" 10
+	create_bundle -L -n bundle1 -v 10 -f /file1,/file2,/file3 "$TEST_NAME"
+
+	sudo sed -i "/^minversion:\\t/d" "$WEBDIR"/10/Manifest.MoM
+
+	create_version "$TEST_NAME" 20 10 staging
+
+	update_bundle "$TEST_NAME" bundle1 --add /file4
+	update_bundle "$TEST_NAME" bundle1 --update /file1
+	update_bundle "$TEST_NAME" bundle1 --delete /file2
```

Comment:
to save a little bit of time you could add the -p flag to the first two update_bundle commands, the third one will take care of re-creating the tar and updating the mom anyway.